department-of-veterans-affairs/abd-vro
# MCP-1640 Not Found Response

# MCP-1640 Not Found Response 

## Description



### What was the problem?


Invalid `claimSubmissionID`s that were passed to the PDF Generator still returned a valid response

### How does this fix it?


Display a proper status for an ID that has never been used before. Specifically `NOT_FOUND` rather than `ERROR` or `IN_PROGRESS`

### Jira Tickets



- [MCP-1640](https://amida.atlassian.net/browse/MCP-1640)

## How to test this PR

- Make a request to fetch a PDF for a random `claimSubmissionID` that hasn't been used previously and the response should say `NOT_FOUND`
